toolium/utils/dataset.py

```diff
@@ -29,6 +29,10 @@
 
 logger = logging.getLogger(__name__)
 
+# Base path for BASE64 and FILE conversions
+base_base64_path = 'resources/'
+base_file_path = 'resources/'
```

Comment:
these default paths change previous behaviour, may be they must be
```
base_base64_path = ''
base_file_path = ''
```

Reply:
Alright, done.